test/cppsuite/src/component/metrics_monitor.cpp

```diff
@@ -124,18 +124,15 @@ metrics_monitor::finish()
         const std::string stat_name = stat->get_name();
 
         /* Append stats to the statistics writer if it needs to be saved. */
-        if (stat->get_save()) {
```

Comment:
The general goal here is to not require callers to know the name, value JSON format.